docs/how-to-guides/data-basics/1-data-basics-setup.md

```diff
@@ -18,7 +18,7 @@ Go ahead and create these files using the following snippets:
 **src/index.js:**
 
 ```js
-import { render } from '@wordpress/element';
+import { render } from 'react-dom';
```

Comment:
There something subtle here is that we should do `import { createRoot } from 'react-dom'` instead of `import { createRoot } from 'react-com/client'`. 

I think that's not great but it's because of how `wp-scripts` works at the moment. Basically it transforms `react-dom` imports to ReactDOM global variable and adds the wp dependency to the php script. And because of React generates the build we use, it works because `createRoot` is actually `ReactDOM.createRoot` in that build.

I think that's very confusing but I don't plan on solving that in this PR.